voluncheer/templates/profiles/profile.html

```diff
@@ -137,6 +137,28 @@ <h2>SAVED OPPORTUNITIES</h2>
                     {% endif %}
                 {% endfor %}
         </div>
+
+        <h2>PAST OPPORTUNITIES</h2>
+        <div class="card" id="saved_opportunities">
+            {% csrf_token %}
+                {% for opportunity in opportunity_lists %}
+                    {% if opportunity.is_archived %}
+                    <article>
+                        <a href="{% url 'update_an_opportunity' opportunity.pk %}"><i id="opportunity-edit-{{opportunity.pk}}" class="fa fa-pen fa-xs edit"></i></a>
```

Comment:
Should we allow users to edit past opportunities? I can think of a reason or two to say yes, but it might be safer to not allow changes after it has passed.

Reply:
I don't think they should be allowed to edit those. Once it has passed it's over; no reason to edit it. The only two things I can see an Organization wanting to do is copying one, or removing it from public site.

Reply:
@EdenWuyifan could you please let us know what the status is on this?

Reply:
Fixed!